runtime/spacetime_nat.c

```diff
@@ -131,7 +129,7 @@ static void reinitialise_free_node_block(void)
 
 enum {
   FEATURE_CALL_COUNTS = 1,
-} features;
+};
```

Comment:
Naming the enum makes it global, which isn't what's wanted (I think)

Reply:
Giving a name to the enum would be written
```
enum features { ... };
```
Here we have an anonymous enum and a global variable `features` of type this enum.  Certainly not what was intended.